README.md

```diff
@@ -29,42 +27,163 @@ to manage.
 
 If you're looking for the previous agent downloader project, it's over [here](https://github.com/OpenIoTHub/ngrok).
 
-# Installation
+# Documentation
+
+A quickstart guide and a full API reference are included in the [ngrok-python API documentation](https://ngrok.github.io/ngrok-python/).
 
-The published library is available on
-[PyPI](https://pypi.org/project/ngrok).
+# Quickstart
+
+1. Install the `ngrok-python` package from [PyPI](https://pypi.org/project/ngrok) using `pip`:
 
 ```shell
 python -m pip install ngrok
 ```
 
-ngrok-python officially supports Python 3.7+.
+2. After you've installed the package, you'll need an Auth Token. Retrieve one on the
```

Comment:
Let's change the verbiage for "Auth Token" to "authtoken" to match how we describe it in the product and documentation. Let's make sure this is reflected throughout the entire readme.

Reply:
Have taken care of this in latest commit.